pkg/util/util.go

```diff
@@ -45,6 +45,23 @@ func removeDuplicatedVolumeMounts(volumeMounts []corev1.VolumeMount) []corev1.Vo
 	return results
 }
 
+func mergeDNSPolicyAndHostNetwork(gPolicy, policy *corev1.DNSPolicy, gHostNetwork, hostNetwork *bool) {
+	if *gPolicy == "" {
+		*gPolicy = *policy
+	}
+
+	if !*gHostNetwork {
+		*gHostNetwork = *hostNetwork
+	}
+}
+
+func setDefaultDNSPolicy(gPolicy *corev1.DNSPolicy) corev1.DNSPolicy {
+	if *gPolicy == "" {
```

Comment:
In the current code, this will never be nil, but if it ever changes, this code will fail. Add a nil check.

Reply:
fine